python/.ci/Jenkinsfile

```diff
@@ -430,6 +430,58 @@ pipeline {
                                         --gcs.to.gcs.sql.query="select * from dataset where sal>1500"'''
                                 }
                             }
+                            stage('PUB/SUB LITE TO BIGTABLE') {
+                                steps{
+                                    sh '''
+                                    
+                                        python3.8 -m pip install --user virtualenv    
+    
+                                        python3.8 -m venv env
+    
+                                        source env/bin/activate
+                                        
+                                        export GCS_STAGING_LOCATION="gs://python-dataproc-templates-temp"
+                                        export JARS="gs://spark-lib/pubsublite/pubsublite-spark-sql-streaming-LATEST-with-dependencies.jar"
+                                        export SKIP_BUILD=true
+
+                                        gcloud pubsub lite-topics publish psltobt \
+                                        --location=us-west1 \
+                                        --message=\'\'\'
+                                        { 
+                                            "rowkey":"rk1",
+                                            "columns": [
+                                                {
+                                                    "columnfamily":"place",
+                                                    "columnname":"city",
+                                                    "columnvalue":"Bangalore"
+                                                },
+                                                {
+                                                    "columnfamily":"place",
+                                                    "columnname":"state",
+                                                    "columnvalue":"Karnataka"
+                                                },
+                                                {
+                                                    "columnfamily":"date",
+                                                    "columnname":"year",
+                                                    "columnvalue":"2023"
+                                                }
+                                            ] 
+                                        }
+                                        \'\'\'
```

Comment:
Move this to the Reset/ Clean Resources stage on the top.

Reply:
[Updated](https://github.com/GoogleCloudPlatform/dataproc-templates/pull/725/commits/0f49e39d066668acc8b09591fafbfa7470416313)